examples/short_example_synthetic_control.py

```diff
@@ -0,0 +1,26 @@
+from cluster_experiments.experiment_analysis import SyntheticControlAnalysis
```

Comment:
some comments about the jupyter:
* point estimate is super good
* power has many mentions to OLS but I don't see it in the plot
* power goes from 0 to 1, can you pick some effects in the middle?